content/mentorship.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Mentorship
+---
+
+# Mentorship Programs
+
+Prometheus offers paid mentorship programs through CNCF mentorship initiatives [https://github.com/cncf/mentorship](https://github.com/cncf/mentorship). Such initiatives create opportunities to people who want to contribute to Prometheus, but have a hard time understanding how to start.
+
```

Comment:
If nothing else, can we link to Thanos mentorship details for now?

Reply:
@bwplotka, are you referring to this [page](https://thanos.io/tip/contributing/mentorship.md/)?